src/components/Section/Foreign/Business/Sponsorship.jsx

```diff
@@ -97,6 +97,7 @@ export default class Sponsorship extends SubsectionElement {
             required={this.props.required}
             scrollIntoView={this.props.scrollIntoView}>
             <SponsorshipItem
+              applicantBirthdate={this.props.applicantBirthdate}
```

Comment:
It doesn't look like `this.props.applicantBirthdate` exists in the case of this component.

Reply:
I'm seeing it in the React console 
![screen shot 2018-11-19 at 1 46 39 pm](https://user-images.githubusercontent.com/1449852/48728774-ec668980-ec03-11e8-88fb-77afd35334f1.png)